### DISCUSSION TITLE
ASReview 0.19 .RIS export: last line in note field deleted ?

### DISCUSSION THREAD

**Discussion by jaohls** [asreview/asreview#943] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hi,

Very happy with the RIS export feature of 0.19, but when importing to EN X9 seems like the last line of the "notes" field gets deleted when the "asreview_relevant" (or irrelevant) tag gets added ? Here is side by side comparison of 2 records, before and after processing through ASReview:

![image](https://user-images.githubusercontent.com/96236396/151333887-f18fc908-3052-4e6d-8f18-61efdfa9f810.png)

Tried a simple workaround by adding a blank line at the end of the field, but didn't work.

**Comment by Rensvandeschoot** [comment, marked as answer]
We created an issue for this topic, see https://github.com/asreview/asreview/issues/944.

**Comment by ottomattas** [comment]
Hey @jaohls, I am trying to replicate this issue - would you be able to share the initial data set you are using? 
It would even help me follow the thread if you could share only the record you are showing in the screenshot and not the whole data set.

**Reply by jaohls** [reply]
Sure, heres the compressed EN library (.enlx) and the RIS format .txt export of same library that i imported to ASReview

[ASR_note_debug.zip](https://github.com/asreview/asreview/files/7992968/ASR_note_debug.zip)

**Comment by ottomattas** [comment]
Hello @jaohls ,
I have tried to replicate the issue you are experiencing. Unfortunately, I was not able to and the exported RIS files look valid to me. See issue #944 for the steps I took.

Is it possible that you have shared a different EN library with us? The screenshot you shared at the start of this discussion do not seem to match with what is in the library.
